code/parse/sexp.h

```diff
@@ -1283,4 +1289,31 @@ int get_effect_from_name(const char* name);
 
 void maybe_write_to_event_log(int result);
 
+#define OSWPT_TYPE_NONE				0
```

Comment:
Reiterating from the other conversation:

> Put the `object_ship_wing_point_team` stuff back where it was, since that should really be confined to the SEXP code. When passing to the ai code, you can use your ship/wing/team enumeration.
> 
> In other words, use oswpt for parsing, then get the ship/wing/team information out of the oswpt struct before you pass it to aicode.